.github/workflows/main.yaml

```diff
@@ -349,3 +349,58 @@ jobs:
         run: |
           export DOCKER_BUILD_AND_SYSTEM_PRUNE="true"
           make image 
+  
+  container_runtime_test:
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        container-runtime: ['cri-o']
+        cases:
```

Comment:
We don't need these cases in container_runtime_test job. Using the default protocal and kubernetes version is enough for now. Please remove these cases and Line 401-402.